src/legal-officer/transaction-protection/LocPublicDataForm.tsx

```diff
@@ -66,16 +66,18 @@ export default function LocPublicDataForm(props: Props) {
                                 message: 'The Data must contain at least 1 character'
                             },
                             maxLength: {
-                                value: 40,
+                                value: 1000,
                                 message: 'The Data Name must contain at most 40 characters'
```

Comment:
The message must be adapted to the new value.

Reply:
Good catch, done!